tests/res_diff_tests/pass_fix_read_compare_errors.ion

```diff
@@ -0,0 +1,22 @@
+{
+ bad: 	{
+	"test": {
+        'ion-java_1': {
+            result: FAIL,
+             read_compare: {
+                errors: ErrorReport::[
+                    {
+                      message: "test",
+                    }
+                ]
+             }
+        },
+        'ion-java_2': {
+            result:FAIL
```

Comment:
Did you mean for this to be `result:PASS`?

Reply:
I added more test cases to separate this.

**This file trying to tes**t: Someone fixes a read_compare error but result is still `FAIL`. For this we should write the diff into the report but not alert users.
**New test case**: Someone fixes a read_compare error and now the second commit's result is `PASS`. same as above.